e2e/e2e-script.sh

```diff
@@ -111,15 +130,16 @@ az vmss create -n ${VMSS_NAME} \
     --instance-count 1 \
     --assign-identity $msiResourceID \
     --image "microsoft-aks:aks:aks-ubuntu-1804-gen2-2021-q2:2021.05.19" \
-    --upgrade-policy-mode Automatic
+    --upgrade-policy-mode Automatic \
+    -ojson
 
 # Get the name of the VM instance to later check with kubectl get nodes
 vmInstanceName=$(az vmss list-instances \
                 -n ${VMSS_NAME} \
-                -g $MC_RESOURCE_GROUP_NAME | \
+                -g $MC_RESOURCE_GROUP_NAME \
+                -ojson | \
                 jq -r '.[].osProfile.computerName'
             )
-export vmInstanceName
```

Comment:
This is consumed by envsubst when generating the pod manifest, so I think the export has to stay.

Reply:
Yes, found out the hard way :sweat_smile: Thanks.